### PR TITLE
Skew function

### PR DESCRIPTION
**Pull request by erap129**
Added skew function (according to issue #536). Very similar to rotation.
Not sure where the documentation is supposed to go (I probably put it in the wrong place)
Opened a new PR instead of the old one that dealt with text only.

**Checklist**:

- [ ] The GitHub pipeline is OK (green), 
      meaning that both `pylint` (static code analyzer) and `black` (code formatter) are happy with the changes of this PR.

- [ ] A unit test is covering the code added / modified by this PR

- [ ] This PR is ready to be merged 

- [ ] In case of a new feature, docstrings have been added, with also some documentation in the `docs/` folder

- [ ] A mention of the change is present in `CHANGELOG.md`

By submitting this pull request, I confirm that my contribution is made under the terms of the [GNU LGPL 3.0 license](https://github.com/PyFPDF/fpdf2/blob/master/LICENSE).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ in order to get warned about deprecated features used in your code.
 
 This can also be enabled programmatically with `warnings.simplefilter('default', DeprecationWarning)`.
 
+## [2.6.2] - not released yet
+### Added
+* `skew` function (similar to `rotation`), along with relevant unit tests and documentation.
```
Comment (Lucas-C) [inline CHANGELOG.md:21]:
```suggestion
* [`FPDF.skew()`](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.skew) - Documentation: [documentation page](https://pyfpdf.github.io/fpdf2/Transformations.html) - thanks to @erap129
```
Also, could you add this in the section dedicated to the upcoming version `2.6.1` please?

File: docs/PageFormatAndOrientation.md [hunk 2]
```diff
@@ -89,3 +89,29 @@ pdf = FPDF()
 pdf.page_mode = "FULL_SCREEN"
 pdf.output("full-screen.pdf")
 ```
+
+
+## skew ##
```
Comment (Lucas-C) [inline docs/PageFormatAndOrientation.md:94]: The documentation and the examples are great! Good job 😃 

Could you move this section to a new `docs/Transformations.md` file please?
We will also add a section dedicated to rotations, as this function is currently not documented.

Also, could you add this new documentation in the `Graphics Content` section in `mkdocs.yml` please?

**Review by Lucas-C — APPROVED** [review]
This is really great!
I'm going to perform a new release once this nice addition has landed on the `master` branch 😃 
Thank you for your contribution @erap129

File: CHANGELOG.md (hunk 1 above)
Comment (erap129) [reply CHANGELOG.md]: done

File: docs/PageFormatAndOrientation.md (hunk 2 above)
Comment (erap129) [reply docs/PageFormatAndOrientation.md]: done

**Comment by Lucas-C** [comment]
Thank you @erap129 for your contribution!
Good job

**Comment by Lucas-C** [comment]
Out of curiosity, what made you want to contribute this PR to fpdf2? 😊

**Comment by erap129** [comment]
> Out of curiosity, what made you want to contribute this PR to fpdf2? blush
I reached you through the up-for-grabs hashtag and it seemed like a cool repo :)

**Comment by Lucas-C** [comment]
> I reached you through the up-for-grabs hashtag and it seemed like a cool repo :)

Ahaha, great! :smile:

**Pull request merged**